### ISSUE TITLE
Improve colour contrast on navigation

### ISSUE THREAD

**Issue by kaitlinnewson** [swcarpentry/website#1255]
### What is the problem?

In the navigation along the top menu, the colour combination of the darker blue background (#4156D9) with black text doesn't pass colour contrast recommendations for accessibility. The current ratio is 3.6:1, and should ideally be around 4.5:1. The lighter blue colour meets these requirements.

One possible fix could be using white text on this background colour instead of black, and it would work for both the darker and light blue backgrounds.

**Comment by maneesha** [comment]
Thanks @kaitlinnewson 
Tagging @elichad and @brynnelliott for feedback.

**Comment by brynnelliott** [comment]
I recommend changing it so that it is the same as carpentries.org.

**Comment by elichad** [comment]
I think it's better to keep the separate colour scheme, so it's clear when the user has moved between carpentries.org and software-carpentry.org. I've opened #1259 to change link text to white when the background is blue, as @kaitlinnewson  suggested.
